qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -324,3 +328,337 @@ def export_to_file(
             file_type=file_type,
             coordinates=coordinates,
         )
+
+
+def convert_dds_to_driven_control(
```

Comment:
Is this a direct copy from the old module?

Reply:
yep. Should have mentioned it in the description.